src/olympia/editors/management/commands/award_post_review_points.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+from django.core.management.base import BaseCommand
+
+import olympia.core.logger
+from olympia import amo
+from olympia.activity.models import ActivityLog
+from olympia.editors.models import ReviewerScore
+
+
+log = olympia.core.logger.getLogger('z.editors.award_post_review_points')
+
+# Note we're going to add to the ReviewerScore this commands adds to be able
+# to find them later.
+MANUAL_NOTE = 'Retroactively awarded for past post/content review approval.'
+
+
+class Command(BaseCommand):
+    help = 'Retroactively award points for past post/content review approvals'
+
+    def handle(self, *args, **options):
+        self.award_all_points_for_action(amo.LOG.APPROVE_CONTENT)
+        self.award_all_points_for_action(amo.LOG.CONFIRM_AUTO_APPROVED)
+
+    def award_all_points_for_action(self, action):
+        if action == amo.LOG.APPROVE_CONTENT:
+            content_review = True
+        else:
+            content_review = False
```

Comment:
imho this could be set explicitly via an argument to the method. But also, nitpick.